pkg/apis/workspaces/v1alpha2/keyed_implementations.go

```diff
@@ -18,3 +19,23 @@ func extractKeys(keyedList interface{}) []Keyed {
 	}
 	return keys
 }
+
+// CheckDuplicateKeys checks if duplicate keys are present in the devfile objects
+func CheckDuplicateKeys(keyedList interface{}) error {
```

Comment:
@amisevsk Because I cannot pass a component/command struct to an interface of type `Keyed` but I can with `interface{}`. The other implementation func `extractKeys(keyedList interface{})` does the same thing. Am I missing something? 🤔

Reply:
Ah you're right, I forgot that you can't cast interfaces like that for Go reasons. Sorry about the confusion!